PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,111 @@
+# Description
+> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context and any links to documentation created.
+> Reference: [PR Submission Guide](/submission_process.md)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+### Dependencies
+> List all libraries and imports utilized, e.g. Ownable.sol from OZ
+
+### Externalities
+> List all external (to this PR) contracts that are interacted with - within Yield ecosystem and without.
+
+> _Note to Reviewer: Examine every component the feature may touch. Then go back through excluded components and ensure they cannot be affected._
+
+### Risk 
+> Highlight areas of risk. E.g. Assembly code blocks, use of unchecked, function call made to an external contract that lies outside of the Yield ecosystem. 
```

Comment:
I think assembly blocks and external contracts should be given extra attention as a matter of practice.  What we want the author of this pr to identify is anything they personally feel was complex or that they are unsure of.
```suggestion
> Highlight areas of risk that are introduced by this pr, or any areas that you would like to be given extra attention.
```